content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
+        
+        The general procedure is as follows:
 
-        \begin{itemize}
-            \item The referee blows the whistle.
-            \item The referee shows his decision for \qty{15}{\second}.
-            \item During that time or in the additional \qty{10}{\second} the robot has time to phrase the referee's decision, e.g., ```Kick-off left team'''. \todo{Or we use a TCP message}
-        \end{itemize}
+        \begin{enumerate}
+            \item The referee blows the whistle once. The referee may use one hand to hold (and blow) the whistle. The referee may instead leave the whistle held in their mouth without their hands.
+            \item The referee selects and indicates a hand-signal. The referee must indicate this signal for \qty{10}{\second}. \todo{reduced this time as 15s is unnecessarily long.}
+            \item During that time or within an additional \qty{10}{\second}, the robot must indicate the hand-signal that it has identified by:
+            \begin{enumerate}
+                \item Using its hand(s) to \emph{mirror} the referee's signal. That is, if the referee given a signals for a decision for the ``red team'' using their right hand, the robot should \emph{mirror} this signal using the robot's \emph{left hand}, as the robot is facing the referee.
+                \item Providing an audio phrase the referee's decision, e.g., ```Goal Red Team'''.  The exact wording is up to the team, but should clearly identify the referee's signal.
+                \item (Optional), At the discretion of the GC/TCM developers, TCP messages may also be sent and interpreted to display the outcome of the robot's decision, however, this is not a requirement for the execution of the challenge. \todo{TW: Allowing this to be optional for now}
+            \end{enumerate}
+            \item The length of time taken for the robot to indicate it's interpretation from the referee blowing the whistle is measured (rounded-up to the nearest second).
+            \item If the robot cannot identify the signal, it should remain motionless and provide no audio output. A robot may continue to pose in the same position as the last signal it identified.
+            \item While not providing a hand-signal or using the whistle, the head referee must keep both hands flat and motionless by their side.
+        \end{enumerate}
 
-        This procedure will be continued for in total four times. Each time, the referee chooses a new decision.
+        This procedure will be repeated \textit{five} times. Each time, the referee chooses a new decision. The referee should choose 2 one-hand static signals, 2 two-hand static signals, and 1 dynamic signal.
```

Comment:
We could allow that the same signal can be chosen twice, so the set of options does not decrease. And your mentioned selection should be randomized.
It would be worth mentioning that the head ref and GC agree before the execution starts on order to be called. So, there are two people to listen to the robot and to support each other

Reply:
yes to all.

Reply:
Thought to my understanding GC isn't required for this challenge (if the TCM packages are not being monitored).